src/ui/package-lock.json

```diff
@@ -8,14 +8,16 @@
         "@chakra-ui/react": "^2.3.5",
         "@emotion/react": "^11.10.4",
         "@emotion/styled": "^11.10.4",
+        "formik": "^2.2.9",
```

Comment:
GitHub is saying there is a diff in this file, but when I copy the raw file and diff in VS Code, I'm not seeing any difference here. Just wanted to write a note here for posterity.

Is it possible that you didn't `git pull` before creating this branch so "resetting" the file actually added the changes from my last PR into your branch history? (No action required, just curious why GitHub is showing a diff).